src/sentry/integrations/msteams/webhook.py

```diff
@@ -211,52 +354,13 @@ def post(self, request: HttpRequest) -> HttpResponse:
         self.verify_webhook_request(request)
 
         data = request.data
-        conversation = data.get("conversation", {})
-        conversation_type = conversation.get("conversationType")
-        event_type = data["type"]
+        raw_event_type = data["type"]
+        event_type = MsTeamsEvents.get_from_value(value=raw_event_type)
 
-        log_params = {
-            "conversation": conversation,
-            "conversation_type": conversation_type,
-            "event_type": event_type,
-        }
+        event_handler_func = self._event_handlers[event_type]
+        response = event_handler_func(request)
```

Comment:
Thankfully here are already existing tests to validate this refactor and code changes